applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -91,6 +92,7 @@ static struct lte_lc_cells_info cell_data = {
 };
 static int ncell_meas_status;
 static char device_id[NRF_CLOUD_CLIENT_ID_MAX_LEN];
+static char cloud_command[16];
```

Comment:
is there a reason this is a string?  i think an enum would be sufficient.
but also, it might not be needed if you remove "DEVICE" message handling.

Reply:
Originaly SLM just intends to support "MODEM" appId, later add "DEVICE" so need some identifier of appId in the same task. This will be removed when we remove "DEVICE" handling